docs/stencil-docs/storefront-customization/ADA compliance-levels.md

```diff
@@ -0,0 +1,301 @@
+| Understandable | On Input             | Consistent Navigation | Change on Request |
+| Understandable | Error Identification | Error Suggestion      |  Help             | 
+
+**Perceivable > Meaningful Sequence: Level A**
+
+Guideline 1.3.2 Meaningful Sequence: When the sequence in which content is presented affects its meaning, a correct reading sequence can be programmatically determined.
+
+**Cornerstone Technique:**
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+The links in the navigation bar form a meaningful sequence. **Search**, **Sign In or  Register**, and **Cart** buttons appear above all other content because many visitors will want to perform these actions.
+ 
+</div>
+</div>
+</div>
+
+**Perceivable > Orientation: Level AA**
+
+Guideline 1.3.4 Orientation: Content does not restrict its view and operation to a single display orientation, such as portrait or landscape, unless a specific display orientation is essential.
+
+**Cornerstone Technique:**
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+Cornerstone is fully responsive, meaning it automatically adapts to your device (desktop, tablet, and mobile).
+ 
+</div>
+</div>
+</div>
+
+**Perceivable > Identify Purpose: Level AAA**
+
+
+Guideline 1.3.5 Identify Purpose: In content implemented using markup languages, the purpose of user interface components, icons, and regions can be programmatically determined.
+
+**Cornerstone Technique:**
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+Cornerstone allows you to add your own icons in the navigation of a site to load symbols and vocabulary familiar to users.
+ 
+</div>
+</div>
+</div>
+
+
+**Operable > Bypass Blocks: Level A**
+
+Guideline 2.4.1 Bypass Blocks: A mechanism is available to bypass blocks of content that are repeated on multiple web pages.
+
+**Cornerstone Technique:**
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+Cornerstone allows you to add a visible 'Skip to Main' link to all pages on your site that sends users to the start of the main content of each page.
+
+Example: /templates/components/common/header.html
+
+```html
+{{lang 'header.skip_to_main'}} {{#if banners.top}}
+{{#each (limit banners.top_metadata 1)}}
+{{{this.content}}}
+{{/each}}
+{{/if}}
+```
+ 
+</div>
+</div>
+</div>
+
+
+**Operable > Multiple Ways: Level AA**
+
+Guideline 2.4.5 Multiple Ways: More than one way is available to locate a web page within a set of web pages except where the web Page is the result of, or a step in, a process.
+
+**Cornerstone Technique:**
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+The search bar is by default in the top right; however, you can receive search results on both the quick-search bar and when you press Enter on the search results page.
```

Comment:
```suggestion
The search bar is in the top right by default; however, you can receive search results on both the quick-search bar and when you press Enter on the search results page.
```

Reply:
Should search results be capitalized (Search Results page)?

Reply:
I took the exact guideline from WCAG.

Reply:
I did not capitalize Search Results page because it is a description of the page, not the heading or title of the page.